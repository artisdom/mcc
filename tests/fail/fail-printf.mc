/* Should be illegal to redefine */
void printf() {}
