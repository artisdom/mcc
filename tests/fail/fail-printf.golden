Error: redeclaration of function printf